rosdep/base.yaml

```diff
@@ -349,9 +359,15 @@ checkinstall:
   debian: [checkinstall]
   ubuntu: [checkinstall]
 chromium-browser:
+  debian:
+    wheezy: [chromium-browser]
```

Comment:
It looks like they've taken over the `chromium` key on debian. https://packages.debian.org/stretch/chromium